pyproject.toml

```diff
@@ -4,6 +4,7 @@ version = "2.1"
 description = "SQLAlchemy for your Django models"
 authors = ["Mikhail Krivushin"]
 license = "BSD 3-Clause \"New\" or \"Revised\" License"
+readme = "README.rst"
```

Comment:
Is `include = ["README.rst"]` missed?

Reply:
I haven’t used include before. I’d be fine adding it, just include a link to the docs so I can understand how poetry thinks about it